openadapt/build_utils.py

```diff
@@ -36,10 +36,11 @@ class RedirectOutput:
     def __enter__(self) -> "RedirectOutput":
         """Redirect stdout and stderr to /dev/null."""
         if is_running_from_executable():
-            null_stream = open(os.devnull, "a")
+            log_file_path = get_root_dir_path() / "data" / "openadapt.log"
```

Comment:
What do you think about adding the date/time to the filename?

Reply:
Makes sense. I was also wondering if we shpuld separate log files by versions?

Reply:
I think as long as the version is logged, date/time in the path is sufficient.

Reply:
Should be easy, and also should help when in cases where the log file gets too large